### ISSUE TITLE
Edit possibility

### ISSUE THREAD

**Issue by we2seek** [nazar-pc/PickMeUp#133]
Thanks for this awesome calendar picker. But I can't figure out how to add possibility to edit selected date from keyboard. Please, show me how. Is it possible at all?

**Comment by nazar-pc** [comment]
Can you elaborate what do you mean by controlling from keyboard?

**Comment by we2seek** [comment]
I mean edit date in input field from keyboard

**Comment by nazar-pc** [comment]
With 3.0 version (latest major) you can type date from keyboard.
I'm sorry, demo was not working, but it is fixed already.
Can you try it now?

**Comment by we2seek** [comment]
Thanks a lot. I'll try it later

**Comment by we2seek** [comment]
Hi! I have some attempts to start with 3.0.2 version. On JSFiddle and locally and catch one kind exception: `pickmeup.min.js:32 Uncaught TypeError: Cannot use 'in' operator to search for 'current' in undefined.`
Would you help me? https://jsfiddle.net/we2seek/712z0c69/17/

**Comment by nazar-pc** [comment]
3.0.3 should work fine with just `pickmeup('.single')`.
Does keyboard work for you eventually?

**Comment by we2seek** [comment]
No, it doesn't. I can change the 1 digit only. Try it on JSFiddle. When input date is 06-09-2016 put cursor to the end of line and press backspace two times

**Comment by nazar-pc** [comment]
Yes, when you change date (when removing one digit) it'll try to parse the whole string and then render it back. Since original string is not valid, it'll be replaced with correct string. Not the most convenient way, but this is how it works right now.
I'll think about introducing some delay or something else to make it behave more natural.
